package.json

```diff
@@ -102,7 +102,7 @@
     "@shapeshiftoss/hdwallet-native": "^1.50.4",
     "@shapeshiftoss/hdwallet-native-vault": "^1.50.4",
     "@shapeshiftoss/hdwallet-shapeshift-multichain": "^1.50.5-alpha.70",
-    "@shapeshiftoss/hdwallet-walletconnectv2": "1.50.8",
+    "@shapeshiftoss/hdwallet-walletconnectv2": "../shapeshiftHdWallet/packages/hdwallet-walletconnectV2",
```

Comment:
Install actual package when https://github.com/shapeshift/hdwallet/pull/635 is published

Reply:
gm https://github.com/shapeshift/web/pull/5382/commits/379dd99b22e62946926978060fa1240891d798d1